tensorboardX/writer.py

```diff
@@ -525,12 +592,12 @@ def add_pr_curve_raw(self, tag, true_positive_counts,
 
         Args:
             tag (string): Data identifier
-            true_positive_counts (torch.Tensor): true positive counts
-            false_positive_counts (torch.Tensor): false positive counts
-            true_negative_counts (torch.Tensor): true negative counts
-            false_negative_counts (torch.Tensor): false negative counts
-            precision (torch.Tensor): precision
-            recall (torch.Tensor): recall
+            true_positive_counts (torch.Tensor, numpy.array, or string/blobname): true positive counts
```

Comment:
How can string/blobname be used?

Reply:
Blob name is a string. Sometimes in Caffe2 user will pass in this blob name, and then we need to call workspace.FetchBlob(blob_name) to get the actual data.

Reply:
I think this should be elaborated. Btw, CI does not test on caffe, right? I had an old caffe2 (0.8.1? not pretty sure) and it fails your code.

Reply:
Just commented this extensively in the _check_caffe2() method. I don't think CI tests on caffe2, I will add in a demo_caffe2.py that will show some example usage in a follow up PR.